sdk/messaging/azeventhubs/README.md

```diff
@@ -98,36 +97,63 @@ When you submit a pull request, a CLA-bot will automatically determine whether y
+[azeventhubs_consumerclient]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs#ConsumerClient
+[azeventhubs_mit_license]: https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/messaging/azeventhubs/LICENSE.txt
+[azeventhubs_repo]: https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/messaging/azeventhubs
+[azeventhubs_repo_main]: https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/messaging/azeventhubs
+[azeventhubs_samples]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs#pkg-examples
+[azeventhubs_pkg_go_docs]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs
+[azeventhubs_processor]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs#Processor
+[azeventhubs_producerclient]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs#ProducerClient
+[azeventhubs_serviceprinciple_consumerclient]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs#example-NewConsumerClient
+[azeventhubs_serviceprinciple_producerclient]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs#example-NewProducerClient
+[coc]: https://opensource.microsoft.com/codeofconduct/
+[coc_faq]: https://opensource.microsoft.com/codeofconduct/faq/
+[coc_contact]: mailto:opencode@microsoft.com
 [default_azure_credential]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity#NewDefaultAzureCredential
-[source]: https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/messaging/azeventhubs
-[godoc]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs
-[godoc_examples]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs#pkg-examples
+[eventhubs_about]: https://docs.microsoft.com/azure/event-hubs/event-hubs-about
+[eventhubs_eventconsumer]: https://docs.microsoft.com/azure/event-hubs/event-hubs-features#event-consumers
+[eventhubs_create_portal]: https://docs.microsoft.com/azure/event-hubs/event-hubs-create
+[eventhubs_create_cli]: https://docs.microsoft.com/azure/event-hubs/event-hubs-quickstart-cli
+[eventhubs_docs]: https://azure.microsoft.com/services/event-hubs/
+[eventhubs_eventpublisher]: https://docs.microsoft.com/azure/event-hubs/event-hubs-features#event-publishers
+[eventhubs_features_terminology]: https://docs.microsoft.com/azure/event-hubs/event-hubs-features
+[eventhubs_namespace]: https://docs.microsoft.com/azure/event-hubs/event-hubs-features#namespace
+[eventhubs_partition]: https://docs.microsoft.com/azure/event-hubs/event-hubs-features#partitions
+[eventhubs_rest_docs]: https://learn.microsoft.com/en-us/rest/api/eventhub/
+[go_install]: https://go.dev/doc/install
+[security_techcenter]: https://www.microsoft.com/msrc/faqs-report-an-issue
 
 ![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-go%2Fsdk%2Fmessaging%2Fazeventhubs%2FREADME.png)
```

Comment:
Curious what this is. It does not exist in the other document that Dan reviewed: https://github.com/Azure/azure-sdk-for-go/pull/20382/commits/3a5bcc1c2cf8b19be181343952f13bc00d4497e9